cocinaborcelle/src/pages/api/cadastrar.ts

```diff
@@ -0,0 +1,55 @@
+import { auth } from "@/../auth/lucia";
+import { Prisma } from "@prisma/client";
+import type { NextApiRequest, NextApiResponse } from "next";
+
+const handler = async (req: NextApiRequest, res: NextApiResponse) => {
+	if (req.method != "POST") {
+		res.status(405);
+		res.end();
+		return;
+	}
+	const { username, password, email } = req.body as {
+		username: string;
+		password: string;
+		email: string;
+	};
+
+	try {
+		const user = await auth.createUser({
+			key: {
+				providerId: "username",
```

Comment:
```suggestion
				providerId: "email",
```

se você tá passando email em providerUserId, então o provedor é o email (isso não muda na funcionalidade da key)